scripts/app_and_key_for_tests.py

```diff
@@ -39,9 +39,8 @@ def _request(method, url, payload={}, headers={}):
             error_body = json.dumps(json.loads(error_body), indent=4)
         except:
             pass
-        print("ERROR after a HTTP request to: %s %s" % (method, full_url))
-        print("%d %s:\n%s" % (e.code, e.reason, error_body))
-        os._exit(1)
+        raise Exception("ERROR after a HTTP request to: %s %s" % (method, full_url)
```

Comment:
For command lines, I prefer to use print/log to stderr and exit code.   
The only difference to the exception is that the stack trace is not printed, usually, it does not bring any additional value to the script output. 

  You could look into `sys.exit("some error message")` it will use logging or  print directly to stderr `print("Hello", file=sys.stderr)`

Reply:
Yup, if the current problem is that the error is written to standard output instead of standard error (and therefore being stored in the shell variable instead of being visible), then it is enough to just write to stderr instead.

Although I generally agree that the stack trace does not provide any extra information here and I avoid them in command line applications, all of the other error handling in this script is done using exceptions (for example look at `run(arguments)`, so it kind of makes sense to use them here as well.

I approve either way :smile:

Reply:
thank you for the feedback, guys! for consistency with the other places, I kept this fix as is.